src/main.tsx

```
@@ -1,10 +1,18 @@
 import { createRoot } from 'react-dom/client'
+
 import './index.css'
 import App from './App.tsx'
 
-createRoot(document.getElementById('root')!).render(
+const container = document.getElementById('root')!;
+
+const root = createRoot(container, SENTRY_CONFIG);
+
+root.render(
   <StrictMode>
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)